src/native-stack/views/NativeStackView.tsx

```diff
@@ -153,6 +153,10 @@ const RouteView = ({
     headerShown,
     hideKeyboardOnSwipe,
     homeIndicatorHidden,
+    sheetAllowedDetents = 'large',
+    sheetLargestUndimmedDetent = 'all',
+    isSheetGrabberVisible = false,
+    sheetCornerRadius = -1.0,
```

Comment:
I wonder whether these default values will work with `react-navigation`, because I remember some shenanigans with default value for `gestureResponseDistance` prop.

Reply:
`react-navigation` will have to add this code by itself since they have their own `NativeStackView`.